clang_format.py

```diff
@@ -19,23 +19,66 @@
 import sys
 import tempfile
 import urllib.request
-
 from pathlib import Path
+from typing import Final, Literal, Mapping, Optional, Sequence, Tuple, Union
 
 # clang-format sha1s were retrieved at Chromium rev
 # 81cc23a856578b149a37dd109b147d8544f9cbd8.
 #
 # https://github.com/chromium/chromium/blob/master/buildtools/linux64/clang-format.sha1
 # https://github.com/chromium/chromium/blob/master/buildtools/mac/clang-format.sha1
 # https://github.com/chromium/chromium/blob/master/buildtools/win/clang-format.exe.sha1
-CLANG_FORMAT_SHAS = {
-    "Linux": "1baf0089e895c989a311b6a38ed94d0e8be4c0a7",
-    "Darwin": "62bde1baa7196ad9df969fc1f06b66360b1a927b",
-    "Windows": "d4afd4eba27022f5f6d518133aebde57281677c9",
+
+CLANG_FORMAT_SHAS: Final[
```

Comment:
Can we write down how to regenerate this list?

Reply:
For getting the list a script is written to download all the sha's from https://commondatastorage.googleapis.com/chromium-clang-format/. and validate them in different os to identify the version and os corresponding to OS. I may have to add the helper script to validate it which i can add at later point of time

Reply:
Cool.  Would you be willing to write a comment (in the Python code, not github) to this effect?

Reply:
Updated comment in clang_format.py.Please review